pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -171,11 +172,21 @@ public void initialize() {
         }
     }
 
+    public boolean isExtensibleLoadManager(){
```

Comment:
Please use the static func, isExtensibleLoadManagerEnabled(), as discussed above.